### PR TITLE
52842_Css user page

### PR DESCRIPTION
**Pull request by fonqhoanq**
## Related Tickets
- [#TicketID](https://edu-redmine.sun-asterisk.vn/issues/52842)

## WHAT (optional)
- Css user page

## Evidence (Screenshot or Video)
Screenshot from 2022-08-26 11-18-01

### CODE REVIEW

**Review by diemntn-0165 — APPROVED** [review]

**Comment by fonqhoanq** [comment]
ready

File: app/helpers/admin/orders_helper.rb [hunk 1]
```diff
@@ -1,11 +1,11 @@
 module Admin::OrdersHelper
   def check_status order
     if order.inactive?
-      return content_tag(:span,"Chưa phê duyệt",class: ["badge","badge-secondary"])
+      return content_tag(:span,I18n.t("admin.orders.order.inactive"),class: ["badge","badge-secondary"])
```
Comment (ngocvt-0484) [inline app/helpers/admin/orders_helper.rb]: bỏ k cần return

File: app/helpers/admin/orders_helper.rb [hunk 2]
```diff
@@ -1,11 +1,11 @@
...
     elsif order.actived? 
-      return content_tag(:span,"Đã phê duyệt",class: ["badge","badge-success"])
+      return content_tag(:span,I18n.t("admin.orders.order.actived"),class: ["badge","badge-success"])
```
Comment (ngocvt-0484) [inline app/helpers/admin/orders_helper.rb]: bỏ k cần return

File: app/helpers/admin/orders_helper.rb (hunk 1 above)
Comment (ngocvt-0484) [inline app/helpers/admin/orders_helper.rb]:
```suggestion
       content_tag(:span, I18n.t("admin.orders.order.inactive"), class: ["badge","badge-secondary"])
```

File: config/locales/vi.yml [hunk 3]
```diff
@@ -249,7 +249,12 @@ vi:
...
       orders_empty: "Danh sách đơn hàng trống"
+      inactive: "Chưa phê duyệt"
+      actived: "Đã phê duyệt"
+      paid: "Đã thanh toán"
     order:
       order_detail: "Xem chi tiết "
     create:
       create_order_fail: "Tạo đơn đặt hàng không thành công"
+      create_order_success: "Tạo đơn đặt hàng thành công"
+
```
Comment (ngocvt-0484) [inline config/locales/vi.yml]: thừa line

**Comment by fonqhoanq** [comment]
ready

**Pull request merged**